app/editor/src/env.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dev": "tno-dev.apps.silver.devops.gov.bc.ca",
```

Comment:
Why not the following so that you don't have to test multiple variable names?
```
{
"dev": ["value1", "value2"],
"test": ["value1", "value2"],
}
```

Reply:
I thought about this initially, and thought we might potentially need the flexibility to use different colors for different urls even for the same environment?

Reply:
If you want that kind of behaviour then you need to have it as part of the config.
```
{
"banner": [
 { "host": "dev.editor.mmia.gov.bc.ca", "color": "#EFEFE1" },
 { "host": "tno-dev.apps.silver.devops.gov.bc.ca", "color": "#EFEFE2" },
 { "host": "test.editor.mmia.gov.bc.ca", "color": "#EFEFE3" },
 { "host": "tno-test.apps.silver.devops.gov.bc.ca", "color": "#EFEFE4" }
}
```

Then the logic in your `Layout.tsx`

```
const color = env.environments.find(e => e.host === window.location.host)
return  color ?? props.theme.css.backgroundColor;
```

You will also need to clean up the sass variables you added as they are no longer needed other than the default colour.

Reply:
Yeah maybe we should just keep it simple as suggested for now :)
Thanks